packages/wallet/src/pages/index.tsx

```diff
@@ -30,22 +30,34 @@ export default function Home({ accounts }: HomeProps) {
       </div>
       <div className="flex w-full flex-col space-y-3 md:max-w-md">
         <div className="mt-2 flex justify-between space-x-2">
-          <Link className="group flex aspect-square basis-1/4 flex-col items-center justify-center rounded-lg border border-green-5 bg-white shadow-md hover:border-green-6">
+          <Link
+            key="send"
```

Comment:
```suggestion
```

We do not need the `key` prop in this case. When we want to generate a list of items from an array the `key` prop is needed.